README.md

```diff
@@ -5,35 +5,45 @@
 
 Start accepting online payments in just a few minutes with our Android SDK. It's quick and easy to integrate, accepts online payments from all major credit cards, and is customizable to your brand.
```

Comment:
Just a suggestion. Can we add:  It's quick and easy to integrate, accepts online payments from all major debit and credit cards? instead of: It's quick and easy to integrate, accepts online payments from all major credit cards.

Reply:
Updated.